assets/wire/components/WireListItem.jsx

```diff
@@ -98,7 +98,7 @@ class WireListItem extends React.Component {
                                 <div className='wire-articles__item__meta-info'>
                                     <span className='bold'>{this.slugline}</span>
                                     <span>{item.source}
-                                        {' // '}<span>{this.wordCount}</span> {gettext('words')}
+                                        {' // '}<span>{this.count}</span> {gettext('words')}
```

Comment:
The variable (this.count) needs initialization (or assignment) - as this count value is not seen on the wire's list item.